gslib/tests/test_cp.py

```diff
@@ -3325,6 +3325,22 @@ def test_kms_key_correctly_applied_to_composite_upload(self):
     with SetBotoConfigForTest([('GSUtil', 'prefer_api', 'json')]):
       self.AssertObjectUsesCMEK(obj_suri, key_fqn)
 
+  @SkipForS3('No composite upload support for S3.')
+  def test_nearline_applied_to_parallel_composite_upload(self):
+    bucket_uri = self.CreateBucket()
+    fpath = self.CreateTempFile(contents=b'abcd')
+    obj_suri = suri(bucket_uri, 'composed')
+
+    with SetBotoConfigForTest([
+        ('GSUtil', 'parallel_composite_upload_threshold', '1'),
+        ('GSUtil', 'parallel_composite_upload_component_size', '1')
+    ]):
+      self.RunGsUtil(['cp', '-s', 'nearline', fpath, obj_suri])
+    stdout = self.RunGsUtil(['ls', '-L', obj_suri], return_stdout=True)
+    self.assertRegexpMatchesWithFlags(stdout,
+                                      r'Storage class:\s+NEARLINE',
```

Comment:
Why use `\s` instead of a regular white space?

Reply:
Becuase the way ls -L works is it tabs NEARLINE over so one space will cause the test to fail.

Reply:
Like inserting the correct number of white spaces. Something like `Storage class:           NEARLINE`

Reply:
After speaking with Nick, changed it to have the correct number of whitespaces.